test/unit/test_sap_cli_transaction.py

```
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+
+import unittest
+from unittest.mock import patch, Mock
```

Flake8 (7.3.0)

[error] 'unittest.mock.patch' imported but unused

(F401)